### DISCUSSION TITLE
Sankey diagram

### DISCUSSION THREAD

**Discussion by Diegomarino80** [team-moeller/better-access-charts#184] [Ideas]
hi,
i think it will be great to have sankey diagram.
And, as a side request, the ability to select the color of every point of a chart

**Comment by team-moeller** [comment]
Hi,
thank you for your idea. I'm with you: a sankey diagram is an interesting chart.
At the moment I have not the necessary time and any plans to make this available in Better Access Charts. So I think it's a good idea to give you some hint to do it on your own.
You can use this guidance from the wiki as a starting point: [add-a-feature](https://github.com/team-moeller/better-access-charts/wiki/add-a-feature).
A Sankey diagram as a plugin for Chart.js is available here: https://github.com/kurkle/chartjs-chart-sankey
I wish you the best of luck with your implementation.

**Reply by Diegomarino80** [reply]
thanks, i kust know VBA and SQL, so i think it will be a bit difficult for me, but i'll give it a try
